modules/tiles/src/tileset/traversers/i3s-tileset-traverser.ts

```diff
@@ -72,8 +72,7 @@ export default class I3STilesetTraverser extends TilesetTraverser {
       ...tileset.loadOptions,
       i3s: {
         ...tileset.loadOptions.i3s,
-        isTileHeader: true,
-        loadContent: false
```

Comment:
Is this a breaking change? Do we need to update docs or upgrade guide?

Reply:
I fount out that loadContent is absolutely useless option in current implementation